### ISSUE TITLE
Pretrained model examples?

### ISSUE THREAD

**Issue by mccartykim** [yanivnik/sinfusion-code#23]
I'm very interested in this paper, but before I dedicate a night of GPU time to this technique, I'd like to play around with a premade checkpoint. Would you consider sharing some of the checkpoints made from the example datasets?

**Comment by yanivnik** [comment]
Certainly. It'll take me about a week since I'm away from my computer right now, but I'll make sure to share some checkpoints once I'm back.
